tests/materialize/materialize-databricks/snapshot.json

```diff
@@ -598,7 +598,7 @@
     {
       "flow_published_at": "1970-01-01T00:00:28Z",
       "id": 1,
-      "unsigned_bigint": 18446744073709552000
+      "unsigned_bigint": 18446744073709551615
```

Comment:
is this meaningful ?

Reply:
I don't totally understand why it was the way it was before. The value for the test fixture is `18446744073709551615` so getting `18446744073709551615` back seems accurate. For some reason it was getting approximated on the older runner. I have always gotten `18446744073709551615` back when running these tests locally (with the same images) so this seems like an improvement.

Some tests (bigquery for example) have always had `18446744073709551615` instead of the approximated version in the snapshot, so I am pretty confident it is just an artifact of how we are extracting the data from the systems for snapshotting, and not something related to the connector.